drivers/Aqara/aqara-presence-sensor/src/init.lua

```diff
@@ -20,32 +20,27 @@ local function status_update(driver, device)
   local conn_info = device:get_field(fields.CONN_INFO)
   if not conn_info then
     log.warn(string.format("refresh : failed to find conn_info, dni = %s", device.device_network_id))
+    return "failed to find conn_info"
```

Comment:
There is no issue with the logic here, but for style purposes, I think that it would be better if this was something more like `return false, "error message"`. The Lua convention that we prefer to use is to return an error in the 2nd return position for fallible functions. And then instead of `return nil` on line 35, you would `return true`.

Reply:
Thank you for letting me know. I modified the code as you said.